### PR TITLE
Add/tiger

### PR DESCRIPTION
**Pull request by baconeta**
Big PR, almost exclusively building the entire predator system and adding them into both the game world and the main scene.

Includes:
- adding all images and setting up animations
- creating a predator spawner (PredatorController.cs)
- Setting up the tiger to be spawned and chase correctly in the right direction, including strafing on the ground to get to the Dodo.
- animation speed adjustment based on the speed of the world at the time the tiger was spawned (ignoring modifiers like pausing for eating)
- some light tidy up and prefab work for the spawner and tiger objects

### CODE REVIEW

File: Assets/Scripts/SaberToothedTiger.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                                 (Time.deltaTime * (_fullSaberSpeed - _worldController.getWorldSpeed()));
+        transform.position += movementVector;
+    }
+
+    private void MoveTowardsDodo()
+    {
+        Vector3 currentPos = transform.position;
+        _directionOfFocus = currentPos - _dodoToChase.transform.position;
+
+        //Is smelled object to Dodo's left or right
```
Comment (TomasMorton) [inline Assets/Scripts/SaberToothedTiger.cs]: Ima guess this is copy pasted from some dodo logic ;)

**Comment by baconeta** [comment]
Had to update since PR to add the ability to kill the tiger

File: Assets/Scripts/SaberToothedTiger.cs (hunk 1 above)
Comment (baconeta) [reply Assets/Scripts/SaberToothedTiger.cs]: NEVER
it's correct naming though :D

File: Assets/Scripts/SaberToothedTiger.cs [hunk 2]
```diff
@@ -27,6 +27,7 @@ public class SaberToothedTiger : Smashable
     // Start is called before the first frame update
     protected override void Start()
     {
+        base.Start();
         _worldController = FindObjectOfType<WorldController>();
```
Comment (TomasMorton) [inline Assets/Scripts/SaberToothedTiger.cs]: Parent sets this already doesn't it?

**Review by TomasMorton — APPROVED** [review]

**Pull request merged**